exist-versioning-release.md

```diff
@@ -208,101 +313,169 @@ You will require a system with:
     What is the new development version for "eXist-db"? (org.exist-db:exist) 5.4.0-SNAPSHOT: :
     ```
 
-6.  Once the prepare process completes you can perform the release. This will upload Maven Artifacts to Maven
-Central (staging), Docker images to Docker Hub, and eXist-db distributions and installer to Github releases:
-    ```
-    $ mvn -Ddocker=true -Dmac-signing=true -Djarsigner.skip=false -Darguments="-Ddocker=true -Dmac-signing=true -Djarsigner.skip=false" release:perform
-    ```
+5. Test the release artefacts as much as possible on different Operating systems (Mac, Windows, Linux).  
+
+   * `$EXIST/exist-distribution/target/` 
+     * eXist-db-<version>.dmg
+     * exist-distribution-eXist-db-<version>.bz2
+     * exist-distribution-eXist-db-<version>-win.zip
+   * `$EXIST/exist-installer-<version>.jar` 
+
+#### Signing Binaries (work in progress)
+
+In order to allow end-users to run the installer as well as the application on macOSX (Catalina) the application needs to be notarized by Apple.
+    
+At this step the application is ready to be notarized and the notarisation ticket must be stapled to the app once that was successful.
+    
+#### Troubleshooting
+
+##### Cleanup dry-run
+
+    mvn \
+      -Pdocker \
+      -Pinstaller \
+      -Pconcurrency-stress-tests \
+      -Pmicro-benchmarks \
+      release:clean
+
+##### Rollback Release
+
+If you need to rollback the `mvn release:prepare` phase:
 
-7.  Update the stable branch (`master`) of eXist-db to reflect the latest release:
+`mvn release:rollback` works but
```

Comment:
I think we should write/discuss a bit more about this. I think that `mvn release:rollback` should never be used in practice...

Reply:
Yes, I would need more information on this.